### PR TITLE
Quick fix to account for recent updates of castor and zdc unpackers

### PR DESCRIPTION
**Pull request by iik1997**
For the Castor online DQM configuration to continue working in recent releases (and during the ongoing PbPb collisions!) this update is needed. Discussed with @BetterWang (Quan Wang), see also discussion in https://github.com/cms-sw/cmssw/pull/12544

### CODE REVIEW

**Comment by iik1997** [comment]
@popovvp, @cbaus, @vanbesien

**Comment by slava77** [comment]
@cmsbuild please test

File: DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -156,85 +156,8 @@
...
-###                           diagnosticPrescaleLS = cms.untracked.int32(-1),
-###                             
-###                           LEDMonitor = cms.untracked.bool(True),
-###                           LEDPerChannel = cms.untracked.bool(True),
-###                           FirstSignalBin = cms.untracked.int32(0),
-###                           LastSignalBin = cms.untracked.int32(9),
-###                           LED_ADC_Thresh = cms.untracked.double(-1000.0)      
-###                           )
+from EventFilter.CastorRawToDigi.CastorRawToDigi_cff import *
+process.castorDigis = castorDigis.clone( UnpackZDC = cms.bool(False) )
```
Comment (slava77) [inline DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py]: why False?

Comment (BetterWang) [inline DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py]: ZDC is monitored in Hcal.

Comment (slava77) [inline DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py]: but this is digis and you can't unpack ZDC from hcalDigis anymore (unless you have a private up to date electronics map).

I recall that HCAL digi monitoring is done on the fly from raw.
So, maybe that's what you mean.

Comment (BetterWang) [inline DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py]: I mean, ZDC DQM is in the hcal_dqm_sourceclient-live_cfg.py.
In castor_dqm_sourceclient-live_cfg.py, there is no need to unpack ZDC, anyway, if you want to put True, it's also fine.

Comment (slava77) [inline DQM/Integration/python/clients/castor_dqm_sourceclient-live_cfg.py]: no need to change. Thanks for the clarification.

**Comment by deguio** [comment]
please submit to 76 and 80 as well.
thanks,
F.

**Comment by iik1997** [comment]
@deguio (sorry for the late reply, I was offline yesterday) 
as fas as I see the unpacker does not include yet the recent modifications implemented now in 75X;
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/EventFilter/CastorRawToDigi/python/CastorRawToDigi_cfi.py
https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/EventFilter/CastorRawToDigi/python/CastorRawToDigi_cfi.py
Let me add also @hvanhaev into the loop as the modifications of the unpacker may have consequences for the planned re-reco of Run2 data in 76X (but again as far as I can see the changes of the unpacker are not yet propagated into 76X)

**Comment by hvanhaev** [comment]
Hi Igor,

You are referring to the inclusion of the ZDC unpacker in our RawToDigi module right?
If this is working fine and producing proper CASTOR and ZDC digi’s, I don’t see any problem for reconstruction…
Regarding 76X: will it ever be used to process HI data with ZDC included? If not, one can think of just leaving the ZDC unpacker modification in 75X.
This is anyway now a temporary fix…

Cheers,
Hans

On 27 Nov 2015, at 08:18, iik1997 <notifications@github.com<mailto:notifications@github.com>> wrote:


@deguio<https://github.com/deguio> (sorry for the late reply, I was offline yesterday)
as fas as I see the unpacker does not include yet the recent modifications implemented now in 75X;
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/EventFilter/CastorRawToDigi/python/CastorRawToDigi_cfi.py
https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/EventFilter/CastorRawToDigi/python/CastorRawToDigi_cfi.py
Let me add also @hvanhaev<https://github.com/hvanhaev> into the loop as the modifications of the unpacker may have consequences for the planned re-reco of Run2 data in 76X (but again as far as I can see the changes of the unpacker are not yet propagated into 76X)

**Comment by slava77** [comment]
For 80X:
can't we make the same cleanup of commented out lines and redo "process.castorDigis=" with just
```diff
 +process.castorDigis = castorDigis.clone() 
```

**Comment by iik1997** [comment]
Yes, fine I will do this then

**Comment by deguio** [comment]
+1

**Comment by slava77** [comment]
merge

**Comment by iik1997** [comment]
Similar PR's for 76X and 80X also done: #12620 and #12621

**Pull request merged**